source/Mission.cpp

```diff
@@ -1455,7 +1456,8 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 		return result;
 	}
 	for(const auto &it : actions)
-		result.actions[it.first] = it.second.Instantiate(subs, sourceSystem, jumps, payload);
+		result.actions[it.first] = it.second.Instantiate(
+			player.Conditions(), subs, sourceSystem, jumps, payload);
```

Comment:
This doesn't seem to exceed 120 characters.
```suggestion
		result.actions[it.first] = it.second.Instantiate(player.Conditions(), subs, sourceSystem, jumps, payload);
```